Samples/AppLifecycle/EnvironmentVariables/cpp-win32-unpackaged/CppWinMainEnv/CppWinMainEnv.cpp

```diff
@@ -0,0 +1,434 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "framework.h"
+#include "CppWinMainEnv.h"
+
+using namespace std;
+using namespace winrt;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Microsoft::Windows::System;
+
+// Windows App SDK version.
+const UINT32 majorMinorVersion{ 0x00010000 };
```

Comment:
Please remove these three lines.  They aren't needed because of the automatic bootstapper initialization.